tensorflow/lite/micro/kernels/xtensa/softmax_int8_int16.cc

```diff
@@ -108,11 +108,11 @@ void* XtensaInitSoftmax(TfLiteContext* context, const char* buffer,
 TfLiteStatus XtensaPrepareSoftmax(TfLiteContext* context, TfLiteNode* node) {
 #if defined(HIFI4) || defined(HIFI4_INTERNAL) || defined(HIFI5)
   return PrepareHifi(context, node);
-#elif defined(VISION_P6)
-  TF_LITE_ENSURE_OK(context, SoftmaxPrepareVision(context, node));
-  return kTfLiteOk;
 #else
   TF_LITE_ENSURE_OK(context, SoftmaxPrepare(context, node));
+#if defined(VISION_P6)
+  TF_LITE_ENSURE_OK(context, SoftmaxPrepareVision(context, node));
```

Comment:
This can just be `return SoftmaxPrepareVision(context, node);`

Reply:
Will look at incorporating this for the next submissions